docs/explanation/security/authentication-and-authorization.md

```diff
@@ -0,0 +1,66 @@
+(authentication-and-authorization)=
+# Authentication and authorization in Landscape
+
+## Authentication methods
+
+Broadly, Landscape users can in be one of two categories:
+
+1. Local users
+2. Externally-authenticated users
+
+Both types of users must receive an invitation to join an account. Users cannot be provisioned automatically.
+
+### Local users
+
+Local users have their email and password stored in the Landscape database. See {ref}`explanation-cryptographic-technology` for details on password storage. Landscape does not offer multi-factor authentication (MFA) for local users.
+
+Local users may only be created when an external identity provider is not configured.
+
+### Integration with external identity providers
+
+Landscape can integrate with
+
+- {ref}`Pluggable authentication modules (PAM) <how-to-external-auth-pam>`
+- {ref}`Active directory <how-to-external-auth-active-directory>`
+- {ref}`OIDC <how-to-external-auth-oidc>`
+
+For any external authentication method, Landscape uses the identity provider for authentication only. Landscape does not integrate with any roles or groups in the external identity provider.
+
+## Authentication model
+
+Prior to 24.04, Landscape used sessions for UI authentication and an access key/secret pair for API authentication. Beginning in 24.04, Landscape introduced JWTs for API and UI authentication.
+
+|         | 24.04                         | 24.10                         | 25.04                         | 25.10                         |
+|---------|-------------------------------|-------------------------------|-------------------------------|-------------------------------|
+| Old UI  | Session only                  | Session <br> or JWT           | Session <br> or JWT           | Session <br> or JWT           |
+| New UI  | JWT only                      | JWT only                      | JWT only                      | JWT only                      |
+| v1 API  | Access key/secret <br> or JWT | Access key/secret <br> or JWT | Access key/secret <br> or JWT | Access key/secret <br> or JWT |
+| v2 API  | JWT only                      | JWT only                      | JWT only                      | JWT only                      |
```

Comment:
A few things:
- Change 24.04 to "24.04 LTS"
- "Old UI" should be "Classic UI" because this is what we call it in the docs, and also it's the main UI for a lot of people still. Normally I wouldn't like "new" UI, but I think it's fine for here since it's not the default UI yet.
- v1 should be Legacy API and v2 should be REST API